Modules/01_oop_and_ood/src/Triangle.ts

```diff
@@ -0,0 +1,44 @@
+import { Shape } from './Shape';
+import { Point } from './Point';
+
+export class Triangle extends Shape {
+
+  constructor(a: Point, b: Point, c: Point, color?: string, filled?: boolean) {
```

Comment:
Again, this is not an overload. By the way, don't you think that 5 arguments are a bit too much? Why not replace the first three args with an arg that takes in an array with exactly three Point instances?

Reply:
![image](https://user-images.githubusercontent.com/3837225/129676860-8de39b41-7e68-4ca6-b9da-8400aa8a4d6a.png)
then it would mean that I need to change spec file, which should not be touched, no? :)

Reply:
Understood, didn't see the spec, you are correct